processing/src/test/java/org/apache/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -29,7 +29,6 @@
 import com.google.common.collect.Lists;
 import com.google.common.collect.Ordering;
 import com.google.common.collect.Sets;
-import com.google.common.util.concurrent.MoreExecutors;
```

Comment:
Were the changes in this file all autoformatter changes? They are hard to follow so I'd rather ask than try to figure that out.

Reply:
Either auto-formatter or `MoreExecutors.sameThreadExecutor()` --> `Execs.directExecutor()`, yes